src/main.js

```diff
@@ -11,6 +11,21 @@ let helpers
 let workerHelpers
 let isConfigAtHomeRoot
 
+const loadDeps = () => {
+  if (!path) {
```

Comment:
This is used down on L321:
```js
    const fileDir = path.dirname(filePath)
```

Reply:
Hmm not sure why Atom search wasn't finding that. It finds it now after searching for fileDir first. Weird. 😕 

I think we can load that one as a const at top level, because Atom probably uses it, along with almost every other package. So it's almost certainly already cached. Then we can drop calling loadDeps in fixJob.

Reply:
We also need `loadDeps` for `workerHelpers.getConfigPath` (for now), as well as `helpers.sendJob`.